web/wp-content/themes/dctx/functions.php

```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).